libopenage/renderer/opengl/uniform_input.cpp

```diff
@@ -50,4 +50,13 @@ GlUniformBufferInput::GlUniformBufferInput(const std::shared_ptr<UniformBuffer>
 	this->update_data.resize(offset);
 }
 
+bool GlUniformInput::is_complete() const {
+  for (const auto& uniform : this->update_offs) {
+    if (!uniform.used) {
+      return false;
+    }
+  }
+  return true;
+}
+
```

Comment:
It looks like this file uses tabs for indenting, while your additions use 2 spaces.